src/common/components/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react'
+import cn from 'classnames'
+
+export type TToggleProps = {
+  classNames?: string
+  selectedClass?: string
+  selected?: boolean
+  toggleHandler: (checked: boolean) => void
+}
+
+const Toggle: React.FC<TToggleProps> = ({
+  children,
+  toggleHandler,
+  classNames = 'w-34px h-5 rounded-full',
+  selectedClass = 'bg-green-68',
+  selected = false,
+}) => {
+  const [checked, setChecked] = useState(selected)
+  return (
+    <div>
+      <label className='flex items-center cursor-pointer'>
+        <div className='relative'>
+          <div
+            className={cn(
+              'block bg-gray-dd flex items-center',
+              classNames,
+              checked && selectedClass,
+            )}
+            onClick={() => {
+              toggleHandler(checked)
+              setChecked(!checked)
```

Comment:
@sanjanadhikari1 is it correct? one is checked, the other is !checked, I think both should be !checked.
Or you can remove setChecked func, we can remove it.

Reply:
yes correct both should be !checked. Thanks for letting me know

Reply:
sounds good